README.md

```diff
@@ -22,6 +22,10 @@ When instantiating the plugin, your `opts` need the correct fields.
   "account": "can be anything; only used for logging",
   "token": "channel name in MQTT server",
   "limit": "amount of negative balance that can exist",
+  "warnLimit": "amount of negative balance before warning to settle",
```

Comment:
I'd argue that it's warning that errors will occur if the balance isn't settled. Could change them to `settleIfOver` or `settleIfUnder` maybe?